index.html

```diff
@@ -113,6 +113,37 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE* Contributor card END ________  -->
 
+
+<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Archita Chaudhary</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://twitter.com/architachaudha5" target="_blank">Your handle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/archita172004" target="_blank">Your handle</a>
+          </p>
+          <p class="about">Hello Everyone, I am Archita Chaudhary a Engineering Student from India. I want to make my career in open-source that is why i am contributing to this project for a nice start \(~*-*~)/</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">https://developer.mozilla.org/en-US/docs/Learn</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">https://roadmap.sh/</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">https://www.freecodecamp.org/</a>
```

Comment:
Please put link of website inside `href`, and the name you want to show in the `a` tag

```suggestion
                <a href="https://developer.mozilla.org/en-US/docs/Learn" target="_blank" title="First Resource">Mozilla Learn web development</a>
              </li>
              <li>
                <a href="..." target="_blank" title="Second Resource">Roadmap.sh</a>
              </li>
              <li>
                <a href="..." target="_blank" title="Third resource">Freecodecamp</a>
```